docs/API.md

```diff
@@ -0,0 +1,6 @@
+## Pessimism API
+
+### Overview
+The Pessimism API is a RESTful HTTP API that allows users to interact with the Pessimism application. The API is built using the [go-chi](https://github.com/go-chi/chi) framework and is served using the native [http package](https://pkg.go.dev/net/http). The API is designed to be modular and extensible, allowing for the addition of new endpoints and functionality with relative ease.
+
+Currently, interactive endpoint documentation is hosted via [Swagger UI](https://swagger.io/tools/swagger-ui/) at [https://pessimism.app/api/docs](https://pessimism.app/api/docs). 
```

Comment:
There's a way to host swagger API through github pages. Maybe worth doing that?

Reply:
https://base-org.github.io/pessimism/